public/static/locales/en/subscriptions.json

```diff
@@ -4,7 +4,8 @@
     "addonCreated": "Add-on created",
     "addonDelete": "Add-on(s) deleted",
     "addonName": "Add-on name",
-    "addons": "Add-ons",
+    "addonRates": "Add-on Rates",
+    "addons": "Add-Ons",
```

Comment:
Not to pile onto the previous comment, but it looks like this string got changed from `Add-ons` to `Add-Ons` (capitalized the O). I can see both being reasonable choices but wanted to mention it in case it wasn't deliberate.

Reply:
Ah, good catch! That wasn't deliberate. I'll fix that.